cmd/loginAuth.go

```diff
@@ -32,27 +20,31 @@ var loginCmd = &cobra.Command{
 	Long:   `Stores tokens in the configuration for the given service.`,
 	PreRun: SetLoggingLevel,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if !pkg.StringSliceContains(allSvcs, svcName) {
-			return fmt.Errorf("service %s not found in supported list of services: %v", svcName, allSvcs)
+		// Handle the legacy way of login via the text file
+		if filePath != "" {
+			return TextFileLogin(cmd, args)
 		}
-
-		logger.Debugf("Saving login credentials for %s using token in file %s...", svcName, filePath)
-		token, err := pkg.ReadFile(filePath)
-		if err != nil {
-			return err
-		}
-		viper.Set(fmt.Sprintf("%s.api.token", svcName), string(token))
-		err = viper.WriteConfig()
-		if err != nil {
-			return err
-		}
-		logger.Tracef("Finished writing credentials for %s using token in file %s...", svcName, filePath)
-		return nil
+		return auth.GetToken()
 	},
 }
 
+func TextFileLogin(cmd *cobra.Command, args []string) error {
+	
+	logger.Debugf("Saving login credentials for %s using token in file %s...", "reservations", filePath)
```

Comment:
Now that "reservations" is hard-coded, I'm not sure the value anymore of using %s in the three strings in this function. It's not a performance issue worth caring about, but it is a bit of a smell.